RetailCoder.VBE/App.cs

```diff
@@ -169,13 +170,29 @@ private void _configService_SettingsChanged(object sender, ConfigurationChangedE
             }
         }
 
+        private void EnsureDirectoriesExist()
+        {
+            try
+            {
+                if (!Directory.Exists(ApplicationConstants.LOG_FOLDER_PATH))
+                {
+                    Directory.CreateDirectory(ApplicationConstants.LOG_FOLDER_PATH);
+                }
+            }
+            catch
+            {
+                //Does this need to display some sort of dialog?
```

Comment:
Make sense.  Just show some sort of dialog and re-throw.  Let it bubble into Extension.cs and shut RD down.